x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -673,8 +677,12 @@ export function registerIndexRoutes({
           headers: { 'content-type': 'application/json' },
         });
       } catch (e) {
-        log.error(`Error simulating inference pipeline: ${JSON.stringify(e)}`);
-        throw e;
+        return createError({
```

Comment:
I wonder whether we should keep logging the error as well 🤔

Reply:
I added the log to see the actual error since the resulting error was saying "see logs" but now that we return the error it seems less helpful IMO. Especially since this is a simulate which is likely to error and be noisy.